### PR TITLE
Fix for fat line pixelation artifacts

### LINKED ISSUE #14197: Pixelated Fat Dashed Lines
Fat dashed lines don't appear to be fully solid on my desktop.

image

I've tested this on two different laptops and fat dashed lines look great so it seems to be machine/browser specific.

I thought this might have been Z fighting but disabling depthTest and removing all other shapes from the scene made no improvement.

### PR DESCRIPTION
**Pull request by WestLangley**
Fixes #14197... At least I am hopeful it will.

### CODE REVIEW

**Comment by WestLangley** [comment]
Live: https://rawgit.com/westlangley/three.js/dev-fat_line_fix/examples/webgl_lines_fat.html

**Comment by pailhead** [comment]
Weird, duplicate of https://github.com/mrdoob/three.js/pull/14275

File: examples/js/lines/LineMaterial.js [hunk 1]
```diff
@@ -205,19 +205,19 @@ THREE.ShaderLib[ 'line' ] = {
 
 			#ifdef USE_DASH
 
-				if ( vUv.y < 0.5 || vUv.y > 0.5 ) discard; // discard endcaps
+				if ( vUv.y < - 1.0 || vUv.y > 1.0 ) discard; // discard endcaps
```
Comment (pailhead) [inline examples/js/lines/LineMaterial.js]: It has a lot of checks though:
```
2x ||
1x ?
4x >
2x if
```

By adding another dimension, you can do it with `bool isCap = vUv.z < 0.5;` and two `if` checks.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**